src/components/ReleaseVersionsRangeFormControl.tsx

```diff
@@ -41,7 +41,7 @@ const ReleaseVersionsRangeFormControl = (props: StackProps) => {
 	const { repository, fromVersion, toVersion } = useComparatorState()
 	const { setFromVersion, setToVersion } = useComparatorUpdater()
 
-	const { data: releases, isLoading } = useReleasesQuery({
+	const { data: releases, isFetching } = useReleasesQuery({
```

Comment:
Same problem we got in the loading spinner never disappearing: we need to use `isFetching` instead of `isLoading`.